### ISSUE TITLE
Example scripts

### ISSUE THREAD

**Issue by abenkovskii** [KSP-KOS/KOS#926]
@erendrake What do you think about shipping some script examples with the mod?

**Comment by erendrake** [comment]
@abenkovskii i dont think that is a bad idea. I wonder what it would take to ship something like the stock game's training scenarios

**Comment by Dunbaratu** [comment]
I'd like to maybe revisit the idea of mission scenarios for career mode.  There's an API for it but I haven't looked into it.  It would be nice if installing kOS and then playing career mode allowed you to learn with baby steps how to do different kOS tasks.  (not the same subject exactly, but I thought I'd mention it).

**Comment by Dunbaratu** [comment]
kslib? I think it should also contain fully executable examples and not just libs.
